tests/publiccloud/run_ltp.pm

```diff
@@ -161,6 +161,31 @@ sub upload_ltp_logs
     die $@ if $@;
 }
 
+sub dump_kernel_config {
+    my ($self, $instance) = @_;
+
+    my $ver_linux_log_path = '/tmp/ver_linux.txt';
+    $instance->run_ssh_command("/opt/ltp/ver_linux > $ver_linux_log_path 2>&1", proceed_on_failure => 1);
```

Comment:
FYI: `/opt/ltp/` should work for you (and you already use `,sudo mv /tmp/publiccloud /opt/ltp/runtest/publiccloud` but if 32 bit package installed, it'd be, see `get_ltproot()`. i.e. running JeOS equivalent of `ltp_cve_m32` or `ltp_syscalls_m32` (LTP compiled as 32bit) would be broken (atm you don't run it).